uamqp/management_link.py

```diff
@@ -156,26 +176,52 @@ def open(self):
         self._request_link.attach()
 
     def execute_operation(
-            self,
-            message,
-            on_execute_operation_complete,
-            timeout=None,
-            operation=None,
-            type=None,
-            locales=None
+        self,
+        message,
+        on_execute_operation_complete,
+        timeout=None,
+        operation=None,
+        type=None,
+        locales=None
```

Comment:
Is this class public? Should these be keyword-only?

Reply:
updated to keyword only